internal/rbd/clone.go

```diff
@@ -190,6 +191,13 @@ func (rv *rbdVolume) doSnapClone(ctx context.Context, parentVol *rbdVolume) erro
 	if err != nil {
 		return err
 	}
+	err = tempClone.unsetAllMetadata(k8s.GetSnapshotMetadataKeys())
+	if err != nil {
+		log.ErrorLog(ctx, "failed to unset volume metadata on temp clone image %s/%s: %v",
```

Comment:
```suggestion
log.ErrorLog(ctx, "failed to unset volume metadata on temp clone image %s %v",
			tempClone, err)
```
tempClone already implements String interface,